app/parse.py

```diff
@@ -1,4 +1,112 @@
-from dataclasses import dataclass
+# import asyncio
```

Comment:
remove this comment

Reply:
> remove this comment

Traceback (most recent call last):
  File "C:\Users\numer\Desktop\Python\py-scrape-quotes\app\parse.py", line 11, in <module>
    @dataclass
     ^^^^^^^^^
NameError: name 'dataclass' is not defined

Reply:
I mean
delete the commented code